src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -2004,6 +2025,11 @@ internal static bool OnServerCertificateCustomValidationCallback(object sender,
                 return errors == SslPolicyErrors.None;
             }
 
+            if (cert is X509Certificate2 cert2 && ExplicitlyTrustedServerCertificates.Contains(cert2.Thumbprint))
```

Comment:
do we call OnServerCertificateCustomValidationCallback in Raven.Server? 

In tests we register something here: https://github.com/ravendb/ravendb/blob/ceb852805e7c8941986874b47336ad129861a671/test/Tests.Infrastructure/TestBase.cs#L137

but if we don't register anything in release verison of server then we don't even reach to line 2028.

Reply:
ok confirmed that - if I comment out 
```
//RequestExecutor.RemoteCertificateValidationCallback += (sender, cert, chain, errors) => true;
```
in TestBase

then test fails. 
```
System.Security.Authentication.AuthenticationException
The remote certificate was rejected by the provided RemoteCertificateValidationCallback.
   at System.Net.Security.SslStream.SendAuthResetSignal(ProtocolToken message, ExceptionDispatchInfo exception)
   at System.Net.Security.SslStream.ForceAuthenticationAsync[TIOAdapter](TIOAdapter adapter, Boolean receiveFirst, Byte[] reAuthenticationData, Boolean isApm)
   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsyncCore(Boolean async, Stream stream, SslClientAuthenticationOptions sslOptions, CancellationToken cancellationToken)
```

Reply:
This remote validator callback seems to be only used by client, so it makes sense. Will test that using 2 separate projects to avoid case of shared configuraiton between client and server.

Reply:
We are using the `RequestExecuter` as well, so that will be possible to run on the server side as well